lib/src/render/color_menu/color_picker.dart

```diff
@@ -105,21 +102,27 @@ class _ColorPickerState extends State<ColorPicker> {
     );
   }
 
-  Widget _buildColorItems(
-    _ColorType type,
-    List<ColorOption> options,
-    String? selectedColor,
-  ) {
+  Widget _buildColorItems(ColorOptionList colorOptionList) {
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       mainAxisAlignment: MainAxisAlignment.start,
-      children: options
-          .map((e) => _buildColorItem(type, e, e.colorHex == selectedColor))
+      children: colorOptionList.colorOptions
+          .map(
+            (e) => _buildColorItem(
+              colorOptionList.onSubmittedAction,
+              e,
+              e.colorHex == colorOptionList.selectedColorHex,
```

Comment:
```suggestion
            (colorOption) => _buildColorItem(
              colorOptionList.onSubmittedAction,
              colorOption,
              colorOption.colorHex == colorOptionList.selectedColorHex,
```
Makes it easier to read, as you don't have to "guess" what `e` refers to, the default `e` refers to `element`, which can be anything.